apps/api/e2e/setup.ts

```diff
@@ -131,6 +131,12 @@ export async function setup() {
     );
   }
 
+  // Extend cache freshness to 10 minutes for all e2e tests to prevent cache expiration issues
```

Comment:
this is kind of interesting, I saw a flaky test failing in CI and I believe the issue was that the test was seeing the price data for the same token come back slightly different between a snapshot and a later calculation of the portfolio balance.  Looking at the logs I can see that the test took over 2 minutes to run on the CI runner (even though it runs in just seconds on my local machine).  My hypothesis is that the change to decrease the cache freshness window from 10 minutes to 1 minute was the source of the failure since now the price was aging out of the cache so the price fetched during the final snapshot diverged slightly from the total pnl calculation during competition close out.  I'm hoping pushing the cache retention back up for tests will keep them stable